tests/Console/Command/ReadmeCommandTest.php

```diff
@@ -28,20 +29,33 @@
 {
     public function testIfReadmeFileIsCorrect()
     {
-        if (!class_exists('Symfony\Component\Console\Output\BufferedOutput')) {
-            $this->markTestSkipped('Unsupported symfony/console version, Symfony\Component\Console\Output\BufferedOutput was added in 2.4.');
-        }
+        $readmeFile = __DIR__.'/../../../README.rst';
+        $this->assertFileExists($readmeFile, sprintf('README file "%s" not found.', $readmeFile)); // switch to `assertFileIsReadable` on PHPUnit6
+        $this->assertTrue(is_readable($readmeFile), sprintf('Cannot not read "%s".', $readmeFile));
+        $this->assertTrue(is_file($readmeFile), sprintf('Expected file "%s".', $readmeFile));
+        $fileContent = file_get_contents($readmeFile);
+        $this->assertInternalType('string', $fileContent, sprintf('Failed to get content of "%s"', $readmeFile));
```

Comment:
Not sure about doing an actual assertion on this as it's a test implementation detail. I would use `fail()` or throw an exception instead.